sw/device/silicon_creator/lib/drivers/BUILD

```diff
@@ -497,9 +496,9 @@ cc_library(
     srcs = ["pinmux.c"],
     hdrs = ["pinmux.h"],
     deps = [
-        "//hw/ip/gpio/data:gpio_c_regs",
-        "//hw/ip/otp_ctrl/data:otp_ctrl_c_regs",
-        "//hw/top_earlgrey/ip_autogen/pinmux:pinmux_c_regs",
+        "//hw/top:gpio_c_regs",
+        "//hw/top:otp_ctrl_c_regs",
+        "//hw/top:pinmux_c_regs",
         "//hw/top_earlgrey/sw/autogen:top_earlgrey",
```

Comment:
Could this top-specific autogen be selected via //hw/top: ?

Reply:
Yes in the multi-top SW PR we have an alias `//hw/top:top_lib` for that. It's just not that in that PR.